scripts/download-resources.mjs

```diff
@@ -168,3 +288,45 @@ export default async function main() {
   }
   await download(kimURL, kimPath, kimSHA[0].split(/\s+/, 1)[0]);
 }
+
+/**
+ * Desired: on Windows, .../bin/kubectl.exe is a copy of .../bin/kuberlr.exe
+ *          elsewhere: .../bin/kubectl is a symlink to .../bin/kuberlr
+ * @param kuberlrPath {string}
+ * @returns {Promise<void>}
+ */
+async function bindKubectlToKuberlr(kuberlrPath) {
+  const binKubectlPath = path.join(binDir, exeName('kubectl'));
+
+  if (onWindows) {
+    await fs.promises.copyFile(kuberlrPath, binKubectlPath);
+
+    return;
+  }
+  try {
+    const binKubectlStat = await fs.promises.lstat(binKubectlPath);
+
+    if (binKubectlStat.isSymbolicLink()) {
+      const actualTarget = await fs.promises.readlink(binKubectlPath);
+
+      if (actualTarget === 'kuberlr') {
+        // The link is already there
+        return;
+      } else {
+        console.log(`Deleting symlink ${ binKubectlPath } unexpectedly pointing to ${ actualTarget }`);
+      }
+    }
+    await fs.promises.rm(binKubectlPath);
+  } catch (_) {
+    // .../bin/kubectl doesn't exist, so there's nothing to clean up
+  }
+
+  const currentDir = process.cwd();
+
+  process.chdir(binDir);
+  try {
+    await fs.promises.symlink('kuberlr', 'kubectl');
+  } finally {
+    process.chdir(currentDir);
+  }
+}
```

Comment:
```suggestion
  await fs.promises.symlink('kuberlr', binKubectlPath);
```

As long as we make a relative link it's okay; we don't need to be in the directory to do so.